src/components/BookList/BookList.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import list from './BookList.module.scss';
+
+const BookList = (props) => {
+  const { booksItem } = props;
+  return (
+    <div className={list.wrapper}>
+      {booksItem.map((book) => (
+        <div className={list.item} key={book.id}>
+          <div>
+            <h3>{book.categories}</h3>
+            <h2>{book.title}</h2>
+            <p>{book.author}</p>
+            <ul>
+              <li>Comments</li>
+              <li>Remove</li>
+              <li>Edit</li>
+            </ul>
+          </div>
+          <div>
+            <i className="fa fa-address-card" aria-hidden="true" />
+            <span>64%</span>
+          </div>
+          <div>
+            <span>CURRENT CHAPTER</span>
+            <p>Introduction</p>
+            <button type="button">UPDATE PROCESS</button>
+          </div>
+        </div>
+      ))}
+    </div>
+  );
+};
+
```

Comment:
- [ ]  As it is a requirement for this project, please add a `Book` component that; 
- Takes the book data as a prop and renders the title and the author
- Has a button for removing books(the button wouldn't remove the books yet, this is just a placeholder for now)
Please checkout the screenshot below for clarifications. 
![booklist](https://user-images.githubusercontent.com/65852621/150001773-07994eac-12f9-4c3c-8f65-a2d25c08d7ac.png)

Reply:
It already has the remove option rendered, and the book and booklist already was rendering title and author.